ports/samd/machine_uart.c

```diff
@@ -119,18 +127,24 @@ STATIC void machine_uart_print(const mp_print_t *print, mp_obj_t self_in, mp_pri
         #if MICROPY_HW_UART_TXBUF
         ", txbuf=%d"
         #endif
+        #if MICROPY_HW_UART_RTSCTS
+        ", flow control=%d"
```

Comment:
Keyword args cannot have spaces in them.  On other ports (eg rp2, stm32) this arg is called `flow`.  But in stm32 it's actually a bit-wise or of CTS and RTS constants.

Reply:
This is no keyword argument. It is just the printout of the object. There is no flow keyword argument in this PR.

Reply:
I understand.  But the point of this print out is that it can be re-evaluated at the REPL to get back the same object.  So the print out should only include kwargs that the constructor supports.

Reply:
Changed.